hyundai_kia_connect_api/KiaUvoAPIUSA.py

```diff
@@ -105,7 +107,7 @@ def api_headers(self) -> dict:
             "content-type": "application/json;charset=UTF-8",
             "accept": "application/json, text/plain, */*",
             "accept-encoding": "gzip, deflate, br",
-            "accept-language": "en-US,en;q=0.9",
+            "accept-language": self.LANGUAGE + ",en-US,en;q=0.9",
```

Comment:
Since regions other than EU haven't been tested I would leave this as is personally.

Reply:
I reverted the changes for non-EU, but still pass the language in the constructor, so this could be used in the future.